compiler/compile/OMRCompilation.hpp

```diff
@@ -965,6 +965,7 @@ class OMR_EXTENSIBLE Compilation
 #endif
 
    // To TransformUtil
+   // @deprecated Use TransformUtil::convertWidthToShift
```

Comment:
Please delete this function outright from OMRCompilation.  While I think there is some benefit of a deprecation period for APIs, this is not such a heavily used API that would benefit from it.